packages/core/core/src/ParcelConfig.js

```diff
@@ -333,10 +333,16 @@ export default class ParcelConfig {
   }
 
   isGlobMatch(filePath: FilePath, pattern: Glob, pipeline?: ?string) {
-    let prefix = pipeline ? `${pipeline}:` : '';
+    let [patternPipeline, patternGlob] = pattern.split(':');
+    if (!patternGlob) {
+      patternGlob = patternPipeline;
+      patternPipeline = null;
+    }
```

Comment:
I guess this be stored in the config and not recalculated every time?

Reply:
Yeah we can also compile the globs to regexes in advance